Riot/Modules/Room/RoomViewController.m

```diff
@@ -7273,6 +7504,54 @@ - (void)documentPickerPresenter:(MXKDocumentPickerPresenter *)presenter didPickD
     }
 }
 
+- (void)sendImage:(NSData *)imageData mimeType:(NSString *)mimeType {
+    // Create or invite again the left member before sending the message in case of a discussion (direct chat)
+    MXWeakify(self);
+    [self createOrRestoreDiscussionIfNeeded:^(BOOL success) {
+        MXStrongifyAndReturnIfNil(self);
+        if (success)
```

Comment:
I'd prefer to display an error if this fails. Applicable for below two methods as well.

Reply:
The error alerts have been added in the last commit for those three methods.